Examples/CustomMachine/jax_wrapper.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019 The Simons Foundation, Inc. - All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+from functools import reduce
+from pickle import dump, load
+import random
+import numpy as _np
+import jax as _jax
+
+import netket
+
+__all__ = ["JAXMachine"]
+
+
+class JAXMachine(netket.machine.CxxMachine):
```

Comment:
can we check that double precision is being actually used?

Reply:
unfortunately the JAX machine is significantly slower (on my machine) than the bare numpy one (at least a factor of 10 or more)... we should look into this and see why this happens